Iterable-iOS-SDK/IterableAPI.m

```diff
@@ -324,22 +372,33 @@ - (void)registerToken:(NSData *)token appName:(NSString *)appName pushServicePla
         return;
     }
     
-    NSDictionary *args = @{
-                           @"email": self.email,
-                           @"device": @{
-                                   @"token": hexToken,
-                                   @"platform": psp,
-                                   @"applicationName": appName,
-                                   @"dataFields": @{
-                                           @"localizedModel": [device localizedModel],
-                                           @"userInterfaceIdiom": [IterableAPI userInterfaceIdiomEnumToString:[device userInterfaceIdiom]],
-                                           @"identifierForVendor": [[device identifierForVendor] UUIDString],
-                                           @"systemName": [device systemName],
-                                           @"systemVersion": [device systemVersion],
-                                           @"model": [device model]
-                                           }
-                                   }
-                           };
+    NSDictionary *deviceDictionary = @{
+                 @"token": hexToken,
+                 @"platform": psp,
+                 @"applicationName": appName,
+                 @"dataFields": @{
+                         @"localizedModel": [device localizedModel],
+                         @"userInterfaceIdiom": [IterableAPI userInterfaceIdiomEnumToString:[device userInterfaceIdiom]],
+                         @"identifierForVendor": [[device identifierForVendor] UUIDString],
+                         @"systemName": [device systemName],
+                         @"systemVersion": [device systemVersion],
+                         @"model": [device model]
+                         }
+                 };
+
+    NSDictionary *args;
```

Comment:
I opted to not merge dictionaries because then you would have to use a NSMutableDictionary in which you can't use the @{} syntax to generate a dictionary.